docs/apache-airflow/installation/installing-from-pypi.rst

```diff
@@ -79,53 +87,64 @@ where:
 - ``AIRFLOW_VERSION`` - Airflow version (e.g. :subst-code:`|version|`) or ``main``, ``2-0``, for latest development version
 - ``PYTHON_VERSION`` Python version e.g. ``3.8``, ``3.9``
 
-There is also a ``constraints-no-providers`` constraint file, which contains just constraints required to
-install Airflow core. This allows to install and upgrade airflow separately and independently from providers.
+However, that should not prevent you from being able to install providers and dependencies that were upgraded
+after the release to keep your installation up-to-dated, even if you do not upgrade airflow core version.
+Installing such dependencies should be done without constraints as a separate pip command.
```

Comment:
What do you think about bolding this line to really make it stand out and hit the point home?

Reply:
Yes. I also rephrased it a little.